services/director-v2/tests/unit/test_comp_tasks_model.py

```diff
@@ -0,0 +1,23 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from pprint import pformat
+from typing import Any, Dict
+
+import pytest
+from pydantic.main import BaseModel
+from simcore_service_director_v2.models.domains.comp_tasks import CompTaskAtDB
+
+
+@pytest.mark.parametrize(
+    "model_cls",
+    (CompTaskAtDB,),
+)
+def test_computation_task_model_examples(
+    model_cls: BaseModel, model_cls_examples: Dict[str, Dict[str, Any]]
+):
+    for name, example in model_cls_examples.items():
+        print(name, ":", pformat(example))
```

Comment:
Add a a test that parses data with ``schema.outputs.defaultValue`` field into ``CompTaskAtdB

Reply:
ah this one disappeared on the train... let me find it again..